src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -17,4 +17,10 @@ public abstract class Command {
      */
     public abstract CommandResult execute(Model model) throws CommandException;
 
+    /**
+     * Returns the type of command.
+     * @return the type of command as a string.
+     */
+    public abstract String commandType();
```

Comment:
Same as earlier, maybe consider renaming this method to be a verb to improve readability.